### ISSUE TITLE
[Central_v2] - Design/Dev Coord - Answer Settings on Create Question

### ISSUE THREAD

**Issue by drewjhart** [rightoneducation/righton-app#1362]

**Comment by rod-hinn** [comment]
So you are saying the answer type (number, text, mathematical expression) and decimal places -- both that info is required for it to work reliability. If that's the case:

This is a slightly longer turn around - let me discuss this with my team since we are actually working on these cards for design system right now.

**Comment by drewjhart** [comment]
Yeah, confirmed that those inputs are req'd in terms of how we handle short answers. Thanks Rod

**Comment by rod-hinn** [comment]
Just doing a clean up of Kanban -- I believe we took care of this? Maybe not on the Kanban but in comments or something. Tell me if it's not!

**Comment by drewjhart** [comment]
@rod-hinn thanks for the flag. I'm not aware of a change for this. Can you point me in the right direction on where your guidance is?

**Comment by rod-hinn** [comment]
[Here](https://www.figma.com/design/PadzU8GqyhFLCjIt7rQkpE/RightOn-?node-id=31357-44517&t=mGiu31KPvVa3qHIw-4).
